doc/nrf/gs_ins_windows.rst

```diff
@@ -46,113 +52,190 @@ To set up the toolchain, complete the following steps:
+   To do so, open a |bash| and enter the following commands (assuming that you have installed the toolchain to |tcfolder_cc|; if not, change the value for GNUARMEMB_TOOLCHAIN_PATH):
 
-     set ZEPHYR_TOOLCHAIN_VARIANT=gnuarmemb
-     set GNUARMEMB_TOOLCHAIN_PATH=c:\gnuarmemb
+   .. parsed-literal::
+      :class: highlight
 
-#. Instead of setting the environment variables every time you open a command prompt, add them as system environment variables or define them in the ``zephyrrc.cmd`` file as described in `Setting up the build environment`_.
+       set ZEPHYR_TOOLCHAIN_VARIANT=gnuarmemb
+       set GNUARMEMB_TOOLCHAIN_PATH=\ |tcfolder|
 
-.. _cloning_the_repositories_win:
+#. Instead of setting the environment variables every time you open a |bash|, |system_vars| define them in the |rcfile| file as described in `Setting up the build environment`_.
 
-.. |bash| replace:: GIT bash
+.. toolchain_end
+
+.. _cloning_the_repositories_win:
 
 .. cloning_start
 
-Cloning the repositories
-************************
+Getting the |NCS| code
+**********************
 
 The |NCS| consists of the following repositories:
 
-* |ncs_zephyr_repo|
 * |ncs_repo|
+* |ncs_zephyr_repo|
 * |ncs_mcuboot_repo|
 * |ncs_nrfxlib_repo|
 
-.. note::
-   The following instructions explain how to clone the repositories to be able to develop your own applications based on the supplied libraries and samples.
-   If you want to actively contribute to the |NCS| development and submit the changes that you make, skip this step and :ref:`fork and clone the repositories <gs_installing_forking>` instead.
+Every |NCS| release consists of a combination of these repositories at different revisions.
+
+The latest state of development is on the master branches of the repositories.
+Note that this might not be a stable state.
+To ensure a usable state, the fw-nrfconnect-nrf defines the compatible states of the other repositories.
+However, this state is not necessarily tested.
+Therefore, unless you are familiar with the development process, you should always work with a specific release of the |NCS|.
+
+To manage the combination of repositories and versions, the |NCS| uses :ref:`zephyr:west`.
+The main repository, fw-nrfconnect-nrf, contains a `West manifest file`_ that defines the versions of all repositories.
+This means that fw-nrfconnect-nrf acts as manifest repository in West (see :ref:`Repository structure <zephyr:west-struct>`), while the other repositories are project repositories.
+To work with a specific release, you initialize West with the corresponding version of the manifest file, and it will check out the correct versions of the project repositories for you.
+In the same way, you can decide to work with the latest state of development by initializing West with the latest manifest file from the master branch of the fw-nrfconnect-nrf repository.
```

Comment:
Again, regardless of what you use during `west init` you can always:

```
cd nrf
git checkout master
west update
```

and that will bring everything to the current master

I would remove the "initializing west" completely, always initialize with master `west init -m <>` with no `--mr` and then instruct people to check out the tag in the nrf repo and run `west update`.

Again, sorry, I did not see that you describe both below so I am amending this comment. I think we need a bit more clarification as mentioned above

Reply:
Also, west is not capitalized (we discussed this upstream)

Reply:
The capitalization of west in the Zephyr docs is not consistent. Will change here, but it should be fixed in Zephyr as well if that has been discussed.

Reply:
I will fix upstream, thanks for letting me know